### PR TITLE
[MRG] add 'info' subcommand

### LINKED ISSUE #165: Add sourmash version output command
`sourmash -v`?
`sourmash version`?

also an `info` or `dbg_info` might be nice to see where it's getting .py and ext module from.

### PR DESCRIPTION
**Pull request by ctb**
Fixes #165.

- [x] Is it mergeable?
- [x] `make test` Did it pass the tests?
- [x] `make coverage` Is the new code covered?
- [x] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [x] Was a spellchecker run on the source code and documentation after
  changes were made?

### CODE REVIEW

**Comment by betatim** [comment]
Add a test that checks that it prints something but has a * for the version? (Unless you can think of an independent way of determining the version to check the command prints the correct one).

Then merge.

**Comment by betatim** [comment]
Added a short test, merge? cc @luizirber @ctb

**Comment by ctb** [comment]
Ready for review @luizirber!

File: sourmash_lib/commands.py [hunk 1]
```diff
@@ -17,6 +17,29 @@
...
+                        help='report versions of khmer and screed')
+    args = parser.parse_args(args)
+
+    from . import VERSION
+    notify('sourmash version {}', VERSION)
+    notify('- loaded from path: {}', os.path.dirname(__file__))
+    notify('')
+
+    if args.verbose:
+        import khmer
```
Comment (luizirber) [inline sourmash_lib/commands.py]: We should probably make `khmer` a [hard dependency in the project][0], or do a try/except block here.

[0]: https://github.com/dib-lab/sourmash/blob/c176cd27f349cfda9e5d2061b100b483804ef4a8/setup.py#L69

Comment (ctb) [inline sourmash_lib/commands.py]: yeah, ok. can we do khmer > 2.0 to avoid 2.0?

Comment (luizirber) [inline sourmash_lib/commands.py]: I think so

Comment (ctb) [inline sourmash_lib/commands.py]: fixed in d1587fc

**Comment by luizirber** [comment]
For some reason khmer is not reporting the version from https://github.com/dib-lab/khmer/archive/master.tar.gz propertly (it works for sourmash and screed, so doesn't seem to be a limitation with versioneer), will investigate.

**Pull request merged**